src/Tribe/Views/V2/iCalendar/Links/Link_Abstract.php

```diff
@@ -261,9 +265,33 @@ public function get_uri( View $view = null ) {
 			return '';
 		}
 
-		$feed_url = str_replace( [ 'http://', 'https://' ], 'webcal://', $feed_url );
+		/**
+		 * Allows filtering of the URL for the subscribe links.
+		 *
+		 * @since TBD
+		 *
+		 * @param string $url The URL for the subscribe link.
```

Comment:
Shouldn't the spacing of these params be flagged by codesniffer...? Looks off to me.